script_editor/app/views/plays/show.html.erb

```diff
@@ -422,26 +457,40 @@
             down = toCut = false
         }
 
+        //next 2 functions actually change the styling of the items
         function cut(element) {
             if (element.className == "word" || element.className == "punc" ||
-                element.className == "stage") {
+                element.className.indexOf("stage") !== -1 || (element.className == "speaker" && speakerCut == true)) {
```

Comment:
includes perhaps?

Reply:
Look at comment on line 487 for more information about .includes()